web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/BeregnOgSimulerRevurderingRoute.kt

```diff
@@ -99,5 +99,9 @@ private fun KunneIkkeBeregneOgSimulereRevurdering.tilResultat(): Resultat {
             "Forventet inntekt kan kun sendes sammen med regulering av grunnbeløp",
             "grunnbelop_forventetinntekt",
         )
+        KunneIkkeBeregneOgSimulereRevurdering.UfullstendigVilkårsvurdering -> BadRequest.errorJson(
```

Comment:
Spiller liten rolle frontend, men er dette mer en IllegalStateException mer enn feil input fra klienten? aka. 500

Reply:
Hmm, det kommer kanskje litt an på hvordan flyt man ser for seg, men kan være tilbøyelig til at det kanskje heller mot 500 i denne flyten.